### ISSUE TITLE
Recent builds are not working with RHEL7

### ISSUE THREAD

**Issue by kneth** [realm/realm-js#5006]
### How frequently does the bug occur?

All the time

### Description

Version 10.21.0 doesn't work on older Linux systems.

### Stacktrace & log output

```shell
node:internal/modules/cjs/loader:1183
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: /lib64/libc.so.6: version `GLIBC_2.25' not found (required by /root/test/node_modules/realm/build/Release/realm.node)
    at Object.Module._extensions..node (node:internal/modules/cjs/loader:1183:18)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at bindings (/root/test/node_modules/bindings/bindings.js:112:48)
    at getRealmConstructor (/root/test/node_modules/realm/lib/index.js:28:37)
    at Object.<anonymous> (/root/test/node_modules/realm/lib/index.js:53:26)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10) {
  code: 'ERR_DLOPEN_FAILED'
}
```

### Can you reproduce the bug?

Yes, always

### Reproduction Steps

`const Realm = require("realm");`

### Version

10.21.0

### What SDK flavour are you using?

Local Database only

### Are you using encryption?

No, not using encryption

### Platform OS and version(s)

RHEL7

### Build environment

Which debugger for React Native: ..

**Comment by crazielips** [comment]
Same problem

**Comment by kneth** [comment]
We have merged a PR which will enable builds for older versions of glibc in order to support Centos/RHEL 7. Binaries will be released from next release.

**Comment by nickbeaulieu** [comment]
Will the fix for this issue be included in version `v10.24.0` or only in version `v11.2.0`? It is only mentioned in the major version 11 [release notes](https://github.com/realm/realm-js/releases/tag/v11.2.0) and not the major version 10 [release notes](https://github.com/realm/realm-js/releases/tag/v10.24.0). 

We had a report of this crash from a few of our users on RHEL.

**Comment by nickbeaulieu** [comment]
Sorry for the delay.

TLDR; `expo 46` uses `react-native < 0.70` so we aren't able to use `realm 11` which requires `react native >=0.70`.

**Background**
We have realm in both a react native app and electron app. The electron app was fine to upgrade to  > 11.2.0 (which we needed for the linux fix). The mobile app has been having issues since upgrading to Expo 47. We started randomly seeing a crash in production builds only after moving from 46 -> 47.

Going back to Expo 46 means we have to use react-native < 0.70 which means we have to use realm < 11. Until we are able to sort out the crashing we'll be sticking with Expo 46.

**Reason**
We _wanted_ to use the same version of the realm SDK in both projects but in reality it seems like it's not that important. We haven't set up sync for the mobile app yet (still using local while under development) so hopefully we don't run into any issues mixing realm versions between the two apps 🤞🏻 

I assume many others are in the same boat of using Expo 46 or less. Not being able to upgrade (or not yet choosing to) to Expo 47 is pretty common since it _just_ came out, but my perception could be off there.
